README_DOCKER.md

```diff
@@ -128,19 +134,19 @@ docker compose run -T --rm django migrate
 
 at any time - still running containers are no problem and usually don't need to be stopped or retarted
 
-### 4. Anything else
+### 6. Anything else
 
-#### 4.1 Stop all running services
+#### 6.1 Stop all running services
 
 To stop all eventually running servics, please execute the command
 
 ```shell
 docker compose stop
 ```
 
-#### 4.2 Modify configuration
+#### 5.2 Modify configuration
```

Comment:
6.2

Reply:
Yeah, I wasn't sure, because starting at point 4 the "heading" (`#` started to be mismatching).

So that's clear now, will make the change.